src/app.js

```diff
@@ -4,44 +4,110 @@ const GitHubApi = require('github');
 const nacl = require('tweetnacl');
 nacl.util = require('tweetnacl-util');
 
-const username = 'yourusername';  // TODO: your GitHub username here
+const username = 'justinborek';  // TODO: your GitHub username here
 const github = new GitHubApi({ debug: true });
 const server = express();
 
+const parser = bodyParser.urlencoded({ extended: false });
+
 // Generate an access token: https://github.com/settings/tokens
 // Set it to be able to create gists
+const GITHUB_TOKEN = 'd41c505e4b808ac55250e7252c1d600c3199eae4';
```

Comment:
This would have been better stored as an environment variable

Reply:
Strongly agreed, and in fact if you've not already you should go into your GitHub settings and revoke this token. Right now people could potentially access it and your account.